samples/TagHelperSample.Web/Views/Home/ConditionalComment.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@addTagHelper "*, TagHelperSample.Web"
+
+<iecondition mode="downlevelRevealed" condition="gt IE 7">
+    <p>
+        Content visible to all browsers newer than Internet Explorer 7. 
```

Comment:
`<p>Content visible to all browsers newer than Internet Explorer 7.</p>` Same format for next one.